src/api/iap/content-types/iap/schema.json

```diff
@@ -26,6 +26,18 @@
       "type": "component",
       "repeatable": true,
       "component": "iap.google-sk-us"
+    },
+    "stripeLegacyIapPrices": {
+      "displayName": "stripeLegacyIapPrices",
+      "type": "component",
+      "repeatable": true,
+      "component": "iap.stripe-legacy-iap-prices"
+    },
+    "stripePlanChoices": {
+      "displayName": "stripePlanChoices",
```

Comment:
`"displayName": "Stripe Plan Choices",`

Reply:
After re-adding the component, the "displayName" property disappeared. I also tried finding a place to update this value, but couldn't find it.